.travis.yml

```diff
@@ -1,8 +1,34 @@
+sudo: required 
+dist: trusty
+
 language: node_js
-node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
-  - "5"
-  - "6"
-  - "7"
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+jobs:
```

Comment:
remove the whole `jobs` section, since we don't actually need to run against mongodb, and replace it with:
```
node_js:
  - 4
  - 6
  - 8
```